src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.ts

```diff
@@ -10,7 +10,8 @@ import { Permission } from '../../models/permissions';
 @Component({
   selector: 'cd-table-actions',
   templateUrl: './table-actions.component.html',
-  styleUrls: ['./table-actions.component.scss']
+  styleUrls: ['./table-actions.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
This seems to cause some disabled buttons are enabled until clicked. Are there any bad implementation in those `disable` functons?

![Peek 2020-10-26 15-38](https://user-images.githubusercontent.com/1691518/97145576-5fd68580-17a1-11eb-898d-7c46f2315033.gif)

Reply:
I have removed this change as it caused side effects. 
Now changes are only in rbd-snapshot components.